docs/architecture/api-credentials-flows.md

```diff
@@ -0,0 +1,18 @@
+# Overview
+
+This document describes how [Application](/docs/terminology.md#Application) provides and updates credentials for [API Definition](/docs/terminology.md#Application-API-Definiton) per [Runtime](/docs/terminology.md#Runtime).
+
+## Details
+This diagram illustrates the API credentials flow in details: 
+
+>**NOTE:** The Application provides [Webhook API](/docs/terminology.md#Application-Webhook-API) where [Management Plane](/docs/terminology.md#Management-Plane) will ask for new credentials for given API Definition.
```

Comment:
>  will ask for new credentials 
requests for providing new credentials
?

Reply:
I would replace first `Note` with a simple sentence. It will look much better than two separate notes, one after another.

```
## Details
This diagram illustrates the API credentials flow in details. The Application provides [Webhook API](/docs/terminology.md#Application-Webhook-API) where [Management Plane](/docs/terminology.md#Management-Plane) will ask for new credentials for given API Definition.
```